source/upgrade.txt

```diff
@@ -48,6 +48,20 @@ changes between the current and upgrade versions. For example, if you
 are upgrading the provider from v8.0 to v8.5, address all breaking changes from
 the version after v8.0, including any listed under v8.5.
 
+.. _entity-framework-breaking-changes-v8.2:
+
+Version 8.2.0 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Version 8.2.0 of the {+provider-short+} does not directly introduce any breaking
+changes. However, the underlying {+csharp-driver-short+} introduces many
```

Comment:
Q; does 8.2 require C# v3.0? if so, might want to be explicit about that

Reply:
I think it comes bundled with it. @damieng can you confirm?

Reply:
It's a direct dependency, so not bundled as such but as soon as your add the MongoDB EF Core Provider NuGet is going to go get the MongoDB C# Driver 3.0 as well.